### ISSUE TITLE
[feat]  support[plugins system] plugins to extend my tauri app, adding functionality, Import external web assets dist ,not merge embedding binary

### ISSUE THREAD

**Issue by cheng990629** [tauri-apps/tauri#5820]
### Describe the problem

support[plugins system] Import external web assets dist ,not merge embedding binary

### Describe the solution you'd like

support[plugins system] Import external web assets dist ,not merge embedding binary

install web plugins system

Like this
<img width="1024" alt="image" src="https://user-images.githubusercontent.com/40415260/207203494-f34778da-9598-4b2e-b936-f8f0267466a5.png">

**Comment by Beanow** [comment]
I think your example is from https://plugins.jetbrains.com/ right?
I'm not sure what you're asking for exactly.

Are you looking for developer / IDE plugins?
Or plugins to extend your app, adding functionality?

**Comment by cheng990629** [comment]
Yes,plugins to extend your app, adding functionality,It can be realized through  Import external web assets dist ,not merge embedding binary
